index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prangon's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="logo-placeholder">
+                <div class="my-logo">
+                    PG
+                </div>
+            </div>
+            <div class="menu">
+                <div class="icon">
+                    <div class="union">
+                        <button class="nav-button"><img src="images/Union.png" alt="menu-image" class="menu-image"></button>
+                    </div>
```

Comment:
- I think it will be better if you wrap your hamburger menu around a `<nav>` tag instead of a div because of semantics reason, and the right position for convention and user experience for mobile devices.

You can simply do this 👇🏽 
```
        <nav class="union">
            <img src="images/Union.png" alt="menu-image" class="menu-image">
          </nav>
```

Also, I don't think it is compulsory to wrap it around a button. Or what do you think?? 🌸